src/components/OpenSource/Contributions.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import styled from 'styled-components'
+import BlueBackground from '../Common/BlueBackground'
+import ContributionsCopy from './ContributionsCopy'
+import { Grid, Row, Col } from '../grid'
+import { Repo } from '../Repo'
+import { generate } from 'shortid'
+import StyledLink from '../Common/StyledLink'
+import Image from '../Common/Image'
+import { BodyPrimary } from '../Typography'
+
+const Wrapper = styled(BlueBackground)`
+  padding: ${props => props.theme.spacing[4]} 0;
+`
+
+const StyledBodyPrimary = styled(BodyPrimary)`
+  padding: ${props => props.theme.spacing[3.5]} 0;
+`
+
+const RepoWrapper = styled.div`
+  padding-bottom: ${props => props.theme.spacing[3]};
+`
+
+const StyledImage = styled(Image)`
+  max-width: 54px;
+`
+
+const Contributions = ({
+  contentfulOpenSourcePage: {
+    contributionsSectionImage,
+    contributionsSectionDescription,
+    contributionsSectionCtaText,
+    contributionsSectionCtaLink,
+    contributionsSectionGithubRepos: repos,
+    contributionsSectionTitleLine1,
+    contributionsSectionTitleLine2,
+    contributionsSectionTitleLine3,
+    openSourceMetaRepoCount,
+    openSourceMetaPullRequestCount
+  }
+}) => {
+  const contributions = {
+    contributionsSectionTitleLine1,
+    contributionsSectionTitleLine2,
+    contributionsSectionTitleLine3,
+    openSourceMetaRepoCount,
+    openSourceMetaPullRequestCount
+  }
+
+  return (
+    <Wrapper>
+      <Grid>
+        <Row>
+          <Col width={[1, 1, 1, 1, 6 / 12]}>
+            <StyledImage image={contributionsSectionImage} />
+            <ContributionsCopy {...contributions} />
+          </Col>
+        </Row>
+        <Row>
+          <Col width={[1, 1, 1, 1, 6 / 12]}>
+            <StyledBodyPrimary textLight>
+              {contributionsSectionDescription}
+            </StyledBodyPrimary>
+          </Col>
+        </Row>
+        <Row>
+          {repos &&
```

Comment:
Any reason why `repos` is the only data from Contentful with the defensive `data &&` on it?

Reply:
Are a lot of these fields not required on the content type? For most cases if they are undefined nothing will happen apart from missing content, it's not very defensive I know but I used it explicitly for the `repos` as if its it null or undefined then `repos.map` will throw